k8s/apiextensions/crds.go

```diff
@@ -87,6 +91,27 @@ func (c *Client) RegisterCRD(crd *apiextensionsv1beta1.CustomResourceDefinition)
 	return nil
 }
 
+// UpdateCRD updates the existing crd
+func (c *Client) UpdateCRD(crd *apiextensionsv1beta1.CustomResourceDefinition) error {
+	if err := c.initClient(); err != nil {
+		return err
+	}
+
+	_, err := c.extension.ApiextensionsV1beta1().CustomResourceDefinitions().Update(crd)
```

Comment:
Return both the updated CRD and error. We do that for all other object's Update apis.